forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -179,3 +259,117 @@ fn build_opts_from_cmd(cmd: &cmd::Deploy) -> pkg::BuildOpts {
         member_filter: pkg::MemberFilter::only_contracts(),
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    fn setup_manifest_files() -> BTreeMap<String, PackageManifestFile> {
+        let mut contract_to_manifest = BTreeMap::default();
+        let manifests_dir = std::env::current_dir().unwrap().join("test/data/manifests");
```

Comment:
It looks like this assumes the location of the `current_dir` which can change depending on from where the user is running `cargo test`. We probably want to use `env!("CARGO_MANIFEST_DIR");` instead? I don't recall if this gives us the package dir or the workspace dir though :thinking:

Reply:
Thanks for this catch!